### ISSUE TITLE
Support for COPC

### ISSUE THREAD

**Issue by jblindsay** [gadomski/las-rs#57]
Thank you for your great work. I am wondering whether there are any plans to support the new COPC compressed lidar format in the future? If so, what are the plans and timeline?

**Comment by gadomski** [comment]
That'd be swell! COPC is just a laz file with extra structure, so you _should_ be able to read COPC files now. In terms of utilizing the range-based request and spatial structure inside COPC, I don't have any plans to add support (this is a personal, unfunded project). But PRs welcome!

**Comment by jblindsay** [comment]
Agreed, reading COPC files is likely supported now, however, I suspect that writing is not. I completely understand the unfunded project limitations. Thanks for the quick response.

**Comment by gadomski** [comment]
For reading, I'll point you to https://github.com/pka/copc-rs

**Comment by jblindsay** [comment]
This is great, thanks for pointing me to this.
